src/walletSdk/Types/horizon.ts

```diff
@@ -38,3 +39,13 @@ export enum HORIZON_ORDER {
   ASC = "asc",
   DESC = "desc",
 }
+
+export type PathPayParams = {
+  destinationAddress: string;
+  sendAsset: StellarAssetId;
+  destAsset: StellarAssetId;
+  sendAmount?: string;
```

Comment:
should we stick with `string` for amount or try using `number` instead for clients' convenience?

Reply:
I see we are using `string` for amounts in other parts of the code, maybe it should be a bigger discussion on which type we prefer and agree on a pattern. Wdyt @piyalbasu @Ifropc @acharb ?

Reply:
I believe most wallets would prefer handling `number` or `BigNumber`, but in case we have a strong reason for keeping `string` that should be fine too. Wallets can always convert the amounts to string whenever necessary

Reply:
It is a breaking change... We can migrate to numbers in 2.0
(I wanted to create Amount wrapper that stores amount as a number and also an asset id)

Reply:
just to give my 2 cents I like using strings inputs for amounts so I don't have to worry about number overflow or big decimal stuff, and letting the sdk handle those complications

but if their apps are already using number/big number everywhere, then I could see that being annoying having to convert

Reply:
We should def think it through in the feature and i'm open to discuss it

Reply:
Though precision is very important, yes. I would like to write something like 
lumens(10) or xlm(5.5) and then pass it where it's needed.